packages/kit/src/runtime/server/cookie.js

```diff
@@ -107,13 +107,23 @@ export function get_cookies(request, url, trailing_slash) {
 		 * @param {import('cookie').CookieSerializeOptions} opts
 		 */
 		set(name, value, opts = {}) {
+			return this.set_internal(name, value, { with_defaults: true, ...opts });
```

Comment:
I don't think we need a `with_defaults` option — we can just pass the defaults from here, no?

```suggestion
			return this.set_internal(name, value, { ...defaults, ...opts });
```

Reply:
Ooh yeah, you're probably right...